packages/frontend/src/ee/features/aiCopilot/components/ChatElements/ToolCalls/AgentToolCalls.tsx

```diff
@@ -38,7 +32,7 @@ const AgentToolCalls = () => {
             >
                 {toolCalls
                     .filter((toolCall) =>
-                        TOOL_NAMES.includes(toolCall.toolName),
+                        TOOL_NAMES.includes(toolCall.toolName as ToolName),
```

Comment:
if ToolName is a zod enum, you can use zod schema's parse and it will handle the type assertion so you don't have to `as`